packages/hardhat/hardhat.config.js

```diff
@@ -50,46 +50,29 @@ module.exports = {
     apiKey: `${process.env.ETHERSCAN_API}`,
   },
   defaults: {
-    // // Local
-    // definitiveMaxTokens: 10000,
-    // definitiveMaxGiftableTokens: 100,
-    // definitiveBatchSize: 50,
-    // definitiveProvenanceHash: '0xabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabca',
-
-    // initialMintPrice: ethers.utils.parseEther('0.2'),
-    // initialEarlyMintPrice: ethers.utils.parseEther('0.015'),
-
-    // // Default hardhat signer[0]
-    // // Will be changed at runtime if targeting a real network.
-    // initialCouponSigner: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
-    // initialUrlChanger: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
-
-    // // Optimistic kovan
-    // definitiveMaxTokens: 10000,
-    // definitiveMaxGiftableTokens: 100,
-    // definitiveBatchSize: 50,
-    // definitiveProvenanceHash: '0xabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabca',
-
-    // initialMintPrice: ethers.utils.parseEther('0.2'),
-    // initialEarlyMintPrice: ethers.utils.parseEther('0.015'),
-
-    // // Default hardhat signer[0]
-    // // Will be changed at runtime if targeting a real network.
-    // initialCouponSigner: '0xA227D2b83aB33384124A75872e1f3D71b6341D0b',
-    // initialUrlChanger: '0xA227D2b83aB33384124A75872e1f3D71b6341D0b',
-
-    // Optimistic mainnet
     definitiveMaxTokens: 10000,
     definitiveMaxGiftableTokens: 100,
     definitiveBatchSize: 50,
-    definitiveProvenanceHash: '0x00000000000000000000000000000000f04a636d10f42ec5a9d4885d30834a70',
+    definitiveProvenanceHash: '0xabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabcabca',
 
     initialMintPrice: ethers.utils.parseEther('0.2'),
     initialEarlyMintPrice: ethers.utils.parseEther('0.015'),
 
     // Default hardhat signer[0]
     // Will be changed at runtime if targeting a real network.
-    initialCouponSigner: '0x8818B7d491aE4F69181Fc62C28a67dDE133b4232',
-    initialUrlChanger: '0xA4403C8388634b9b01b70763e2C5C44FEC138250',
+    initialCouponSigner: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+    initialUrlChanger: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+  },
+  overrides: {
```

Comment:
Definitely like the cleanup!

However, overrides in ethers refers to tx data stuff. Maybe it's slightly less confusing to put these as sub-objects of the deafults object, or inside networj.defaults?

Reply:
Tried that! But it throws an error because hardhat validates the network objects

Reply:
Done, changed `defaults` to `defaultParameters` and `overrides` to `customParameters`